extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php

```diff
@@ -122,7 +122,10 @@ public function track($amgId, $gracenoteId, Title $title) {
 
 		wfDebug(__METHOD__ . ': ' . json_encode($data) . "\n");
 
-		$resp = Http::post($url, ['postData' => $data]);
+		$resp = Http::post($url, [
```

Comment:
We perform more internal requests than external ones (by both number of requests and places in the code that make them). I was thinking about introducing `ExternalHttp` helper class with `get` and `post` methods. With the "HTTP headers for request tracing" we have a second use case for this helper class. Let's implement it.